src/modality-specific-files/positron-emission-tomography.md

```diff
@@ -73,10 +73,28 @@ and a guide for using macros can be found at
    suffixes=["pet", "events", "physio", "stim"])
 }}
 
-PET data MUST be stored in the `pet` directory.
-PET imaging data SHOULD be stored in 4D (or 3D, if only one volume was acquired)
-NIfTI files with the `_pet` suffix.
-Volumes MUST be stored in chronological order (the order they were acquired in).
+### PET data format
+
+All PET imaging data MUST be stored using the
+[NIfTI-1 or NIFTI-2](https://nifti.nimh.nih.gov/) file format (`.nii`).
+The ANALYZE-7.5 file format (`.hdr`/`.img`), despite being very similar to NIFTI-1,
+is not allowed.
+Imaging data SHOULD be converted to the NIfTI format using a tool that provides
+as much of the NIfTI header information (such as orientation and slice timing
+information) as possible.
+
+Dynamic (multi-volume) PET imaging data SHOULD be stored in 4D,
+in chronological order (the order they were acquired in).
+
+Due to the important size of PET imaging data, we RECOMMEND using compressed
```

Comment:
we do not actually officialy recommend (meaning that the validator would throw a warning if people use `.nii` and not `.nii.gz`)

Maybe "we suggest" or "we enourage".

Reply:
Recommendation was carried from [common principles](https://bids-specification.readthedocs.io/en/stable/common-principles.html#imaging-files):

>  We RECOMMEND using compressed NIfTI files